src/src/com/microsoft/aad/adal/SerializedTokenCacheItem.java

```diff
@@ -0,0 +1,40 @@
+package com.microsoft.aad.adal;
+
+import com.google.gson.annotations.SerializedName;
+
+final class SerializedTokenCacheItem {
+
+    private static final String TAG = SerializedTokenCacheItem.class.getSimpleName();
+
+    @SerializedName("authority")
+    private final String mAuthority;
+
+    @SerializedName("refreshToken")
+    private final String mRefreshToken;
+
+    @SerializedName("idToken")
+    private final String mIdToken;
+
+    @SerializedName("familyClientId")
+    private final String mFamilyClientId;
+
+    SerializedTokenCacheItem(final TokenCacheItem tokenCacheItem) {
+        this.mAuthority = tokenCacheItem.getAuthority();
+        this.mRefreshToken = tokenCacheItem.getRefreshToken();
+        this.mIdToken = tokenCacheItem.getRawIdToken();
```

Comment:
We are trying to keep a clean separation between serialization formats.  There's no need to carry the empty fields, so we leave them behind.  We can always add things back in later as needed.  

However, I think we should do what @agamemnum suggested and turn this into a serialization type adapter.